dir.targets

```diff
@@ -45,4 +45,15 @@
     <!-- We set this property to avoid MSBuild errors regarding not setting TargetFrameworkProfile (see above line) -->
     <PortableNuGetMode>true</PortableNuGetMode>
   </PropertyGroup>
+
+  <!--
+    add the test-runtime project.json to the list of supplemental test data so it will get included
+    in the test archive; the stress test build system depends on it when building test mixes.
+  -->
+  <ItemGroup Condition="'$(IsTestProject)'=='true'">
```

Comment:
Is it reasonable to expect that level of uniformity?  E.g. some repos don't have this today and perhaps they never will.  Or a possible case where there's multiple files to include in this way?